README.md

```diff
@@ -1,5 +1,7 @@
 # DevOps-Apprenticeship-Notes
 
+* take a vdio at our bitwise industries standards and practices here  https://www.youtube.com/watch?v=YcJ9IeukJL8: https://www.youtube.com/watch?v=SLB_c_ayRMo: https://www.youtube.com/watch?v=gxPykhP
```

Comment:
Can we take a look here and correct the spelling errors? I would also recommend when adding links to markdown, to make each link its own line and give a small description before.